src/tiledquick/qml/main.qml

```diff
@@ -109,11 +138,7 @@ ApplicationWindow {
 
     Item {
         id: mapView
-
-        scale: 1 / Screen.devicePixelRatio
-        width: parent.width * Screen.devicePixelRatio
-        height: parent.height * Screen.devicePixelRatio
```

Comment:
Actually, this code is necessary precisely because Qt Quick is scaling. It undoes that scaling for the map view, where I explicitly want 1 pixel to be 1 pixel at 100% zoom. We don't want to scale up the user's graphics, just the UI.

Reply:
Ah right. I forgot about that.